### PR TITLE
Test alert and comment models

### PR DESCRIPTION
**Pull request by nadavsofi**

### CODE REVIEW

File: client/.gitignore [hunk 1]
```diff
@@ -8,3 +8,4 @@ config/local.json
 .DS_Store
 cypress/videos/
 cypress/screenshots/
+http-client/
```
Comment (jbesraa) [inline client/.gitignore:11]: is http-client part of the project?

File: server/package.json [hunk 2]
```diff
@@ -63,7 +63,8 @@
         "build": "./node_modules/react-scripts/bin/react-scripts.js build",
         "serve": "node bin/serve",
         "lint": "./node_modules/.bin/eslint . --fix",
-        "test": "NODE_ENV=test ./node_modules/.bin/mocha ./tests/integration/* ./tests/unit/* --exit --recursive --require ./tests/setup.js"
+        "test": "NODE_ENV=test ./node_modules/.bin/mocha ./tests/integration/* ./tests/unit/* --exit --recursive --timeout 8000 --require ./tests/setup.js",
```
Comment (jbesraa) [inline server/package.json:66]: lets increase the timeout to 10000

File: server/package.json [hunk 3]
```diff
@@ -63,7 +63,8 @@
...
+        "watch": "nodemon bin/api"
```
Comment (jbesraa) [inline server/package.json:67]: use project path, so: ./node_modules/.bin/nodemon bin/api

File: server/tests/docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,24 @@
+version: '3.6'
```
Comment (jbesraa) [inline server/tests/docker-compose.yml:1]: I would add some text on the of the top of this file to explain what its used for..

**Review by jbesraa — CHANGES_REQUESTED** [review]

File: client/.gitignore (hunk 1 above)
Comment (nadavsofi) [reply client/.gitignore:11]: not anymore

**Comment by jbesraa** [comment]
@nadavsofi  can you please update this PR?

**Review by jbesraa — APPROVED** [review]

**Comment by jbesraa** [comment]
@florpor this is good to go. nothing special here

**Pull request merged**